### ISSUE TITLE
Question about new release

### ISSUE THREAD

**Issue by security-companion** [Neo23x0/Loki#259]
Hi,
I read that loki is not supported anymore and that you recommend using Thor Lite instead.

Nevertheless I would like to ask if it is possible to make a new release of loki so that the latest code changes can be used via an installer (I know that it however possible to make a git clone and use the latest code from there but for that you first need to have python installed).

The reason why I ask is that when running a scan with Thor Lite the html report says that Thor Lite is only allowed for non-commercial use.
So from my understanding the only one of your free scanners that can be used in a commercial environment is loki.

Greetings
security-companion

**Comment by Neo23x0** [comment]
I don't want the scanner to print a warning every time it skips a file for any reason. 
We could talk about "notice" level messages, but skipping a file is expected behaviour. I don't want it to print "Warnings" when the scanner works as expected.

**Comment by security-companion** [comment]
Thanks for mering my pull request.
I had chosen info as this is also set for the other "Skipping file"-message (Skipping file due to fast scan mode)
We can also change the "Skipping file due to file size"-message from info to notice, see #262

**Comment by Neo23x0** [comment]
Oh, you say, you've chosen "Info", but I saw a "Warning" in the code of the PR: 

https://github.com/security-companion/Loki/commit/9b9656bde3a30ba41f73c4cd8726db891cac4bf9#diff-6f22c54946bc6e83f9850b3cdef74f5040e6b8f0d6e5d46980b46cb0674dc00fR372

"Info" level would be best.

**Comment by security-companion** [comment]
Yes, originally it was warning but one day after the commit you reference I changed it in https://github.com/security-companion/Loki/commit/191086baf3e8d3073aaac6679d9044127f4a27b7 from warning to info. This is what then got merged into trunk in pull request https://github.com/Neo23x0/Loki/pull/239/files

Currently we have info in the main-branch, see https://github.com/security-companion/Loki/blob/88c8d3e710cbc35ad7f282c69d4f572bce4015a4/loki.py#L372
